ee-9/pom.xml

```diff
@@ -1536,6 +1536,54 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.narayana.rts</groupId>
+                <artifactId>restat-bridge-jakarta</artifactId>
+                <version>${version.org.jboss.narayana}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.jboss.narayana.rts</groupId>
+                <artifactId>restat-api-jakarta</artifactId>
```

Comment:
We have a license entry for the nonnative one on `ee-feature-pack-common-licenses.xml`, so I guess we need a new entry for the Jakarta native one.